Hedvig/Generic/Viewables/CircleLabelSmall.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  CircleLabelSmall.swift
+//  Hedvig
+//
+//  Created by Sam Pettersson on 2019-01-04.
+//  Copyright © 2019 Hedvig AB. All rights reserved.
+//
+
+import Flow
+import Form
+import Foundation
+import UIKit
+
+struct CircleLabelSmall {
+    let labelText: DynamicString
+    let color: String
+
+    init(labelText: DynamicString, color: String) {
+        self.labelText = labelText
+        self.color = color
+    }
+}
+
+extension CircleLabelSmall: Viewable {
+    func materialize(events: ViewableEvents) -> (UIView, Disposable) {
+        let circleView = UIView()
+        let bag = DisposeBag()
+
+        let labelsContainer = CenterAllStackView()
+        labelsContainer.axis = .vertical
+
+        let titleLabel = UILabel()
+        titleLabel.font = HedvigFonts.circularStdBook?.withSize(14)
+        titleLabel.lineBreakMode = .byWordWrapping
+        titleLabel.textAlignment = .center
+        titleLabel.numberOfLines = 0
+        titleLabel.adjustsFontSizeToFitWidth = true
+        bag += titleLabel.setDynamicText(labelText)
+
+        circleView.layer.shadowOpacity = 0.2
+        circleView.layer.shadowOffset = CGSize(width: 10, height: 10)
+        circleView.layer.shadowRadius = 16
+        circleView.layer.shadowColor = UIColor.darkGray.cgColor
```

Comment:
You should always specify a shadow path due to performance savings.

This should work:
```swift
circleView.shadowPath = UIBezierPath(roundedRect: bubbleView.bounds, cornerRadius: circleView.frame.height * 0.5)
```

Add it to the `didLayoutSignal.onValue` on L62